src/test/java/tests/GmailTest.java

```diff
@@ -0,0 +1,107 @@
+package tests;
+
+import by.issoft.config.WebDriverSingleton;
+import by.issoft.po.*;
+import org.testng.annotations.*;
+
+import java.net.MalformedURLException;
+
+import static org.testng.Assert.assertTrue;
+
+@Listeners(Listener.class)
+class GmailTest {
+
+	private final String SENDER_EMAIL = "seleniumtests10@gmail.com";
+	private final String SENDER_PASS = "060788avavav";
+	private final String SENDER_USERNAME = "Selenium Selenium";
+	private final String RECEIVER_EMAIL = "seleniumtests30@gmail.com";
+	private final String RECEIVER_PASS = "060788avavav";
+	private final String RECEIVER_USERNAME = "Selenium";
+	private final String EMAIL_TITLE = "Test email title";
+
+	private static LoginPage loginPage;
+	private static LoggedInPage loggedInPage;
+	private static InboxEmailsPage inboxEmailsPage;
+	private static SentEmailsPage sentEmailsPage;
+	private static TrashPage trashPage;
+	private static NavigationPage navigationPage;
+	private static ChangeDeleteAccountPage changeDeleteAccountPage;
+
+	public GmailTest() throws MalformedURLException {
+		loginPage = new LoginPage();
+		loggedInPage = new LoggedInPage();
+		inboxEmailsPage = new InboxEmailsPage();
+		sentEmailsPage = new SentEmailsPage();
+		trashPage = new TrashPage();
+		navigationPage = new NavigationPage();
+		changeDeleteAccountPage = new ChangeDeleteAccountPage();
+	}
+
+	@BeforeClass(alwaysRun = true)
+	public static void setUp() throws MalformedURLException {
+		WebDriverSingleton.getInstance().goToPage("https://www.google.com/gmail");
+	}
+
+	@AfterClass(alwaysRun = true)
+	public static void tearDown() throws MalformedURLException {
+		WebDriverSingleton.getInstance().closeWebDriver();
+	}
+
+	@DataProvider(name = "loginData")
```

Comment:
I prefer to create separate class for data providers (they can contain readers from db/api/etc.)